test/test_pegen.py

```diff
@@ -367,3 +368,108 @@ def test_cut():
     assert node == [TokenInfo(OP, string='(', start=(1, 0), end=(1, 1), line='(1)'),
                     [TokenInfo(NUMBER, string='1', start=(1, 1), end=(1, 2), line='(1)')],
                     TokenInfo(OP, string=')', start=(1, 2), end=(1, 3), line='(1)')]
+
+
+class TestGrammarVisitor:
```

Comment:
Why not continue the pytest style test functions, without the test case class?

Reply:
I can remove the class if you prefer. I normally use classes (still pytest-style classes with no extra inheritance or anything) to group related tests together in the same file, so is a bit more organized and then I can select to collapse the things that I am not looking at.